spec/factories/collection_user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :collection_user do
```

Comment:
Is this a useful factory? I'd rather expect to see the association between a user and collection be created in the test setup explicitly.

Reply:
I use it in `collection_user_spec.rb` to validate the presence of validations. I think it makes sense and is useful, even if only for that one case.